src/test/java/seedu/linkedout/model/applicant/ApplicantTest.java

```diff
@@ -45,10 +45,10 @@ public void isSameApplicant() {
         Applicant editedBob = new ApplicantBuilder(BOB).withName(VALID_NAME_BOB.toLowerCase()).build();
         assertFalse(BOB.isSameApplicant(editedBob));
 
-        // name has trailing spaces, all other attributes same -> returns false
-        String nameWithTrailingSpaces = VALID_NAME_BOB + " ";
-        editedBob = new ApplicantBuilder(BOB).withName(nameWithTrailingSpaces).build();
-        assertFalse(BOB.isSameApplicant(editedBob));
+        // name has multiple whitespaces, all other attributes same -> returns true
+        String nameWithMultipleSpaces = VALID_NAME_BOB + " ";
```

Comment:
This part seems to be single space instead of multiple space correct me if i'm wrong

Reply:
The implementation removes all 'extra' whitespaces, be it before, between or after the words

Reply:
This does not seem to be multiple spaces, instead this is just the trailing whitespace but instead they are now the expected result is true. 

Try to add a second, more comprehensive test (include a scenario where there is a multiple whitespace) Right now `VALID_NAME_BOB = Bob Choo`, try a version with `INVALID_NAME_BOB = VALID_NAME_BOB.replaceAll(" ", "  ")`